cmdb/importer/importer_base.py

```diff
@@ -167,7 +167,7 @@ def _import(self, import_objects: list) -> ImporterObjectResponse:
                     success_imports.append(ImportSuccessMessage(public_id=current_public_id, obj=current_import_object))
             else:
                 try:
-                    self.object_manager.delete_object(current_public_id, self.request_user, AccessControlPermission.DELETE)
+                    self.object_manager.delete_object(current_public_id, self.request_user)
```

Comment:
Why did you remove the permission? So the validation is not executed, right? How could this have worked in the test?

Reply:
We can discuss that. Based on the original logic, it is not checked whether the user is allowed to delete objects. It follows that the check for ACLs is also not necessary. Then the question arises. Do the rights 'Delete objects' and / or the ACLs 'Delete' have to be available for importing objects? And why are the objects deleted during import?